README.md

```diff
@@ -184,6 +185,87 @@ $ remme public-key get-list \
 }
 ```
 
+### Transaction
+
+Get list of transaction — ``remme transaction get-list``:
```

Comment:
`Get list of transaction` > `Get a list of transactions`

Reply:
@dmytrostriletskyi, done.